seed/serializers/inventory_document.py

```diff
@@ -11,10 +11,41 @@
 from seed.models import InventoryDocument
 from seed.serializers.base import ChoiceField
 
+import logging
+import boto3
+from botocore.exceptions import ClientError
+from django.conf import settings
+
+
+def create_presigned_url(bucket_name, object_name, expiration=604800):
```

Comment:
Is the expiration time supposed to be that large? I feel like `expiration=3600` would be fine here too!